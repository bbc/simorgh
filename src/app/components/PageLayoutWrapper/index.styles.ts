import { css, Theme } from '@emotion/react';

export default {
  wrapper: ({ palette }: Theme) =>
    css({
      minHeight: '100vh',
      display: 'flex',
      flexDirection: 'column',
      justifyContent: 'space-between',
      backgroundColor: palette.GHOST,
    }),
  content: css({
    flexGrow: 1,
  }),
};
