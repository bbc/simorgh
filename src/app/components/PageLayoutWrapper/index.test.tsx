import React from 'react';

import { render } from '../react-testing-library-with-providers';
import PageLayoutWrapper from '.';

global.performance.getEntriesByName = jest.fn(() => []);

describe('PageLayoutWrapper', () => {
  it('should render default page wrapper with children', async () => {
    const { container } = render(
      <PageLayoutWrapper
        // @ts-expect-error - metadata type is mocked for test purposes
        pageData={{ metadata: { type: 'test-page-type' } }}
        status={200}
      >
        <h2>Child element</h2>
      </PageLayoutWrapper>,
    );

    expect(container).toMatchSnapshot();
  });
});
