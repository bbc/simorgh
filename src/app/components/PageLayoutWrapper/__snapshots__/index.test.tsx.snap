// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLayoutWrapper should render default page wrapper with children 1`] = `
.emotion-0 {
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  background-color: #FDFDFD;
}

.emotion-1 {
  position: fixed;
  bottom: 0;
  left: 0;
  width: 100%;
  z-index: 2147483647;
}

.emotion-3 {
  font-family: ReithSans,Helvetica,Arial,sans-serif;
  font-weight: 400;
  font-style: normal;
  background-color: #323232;
  border-top: solid 0.0625rem transparent;
}

.emotion-5 {
  max-width: 37.4375rem;
  margin: 0 auto;
}

.emotion-5 a {
  color: #F6A21D;
  -webkit-text-decoration: none;
  text-decoration: none;
  border-bottom: solid 0.0625rem #AEAEB5;
}

.emotion-5 a:focus,
.emotion-5 a:hover {
  color: #222222;
  background-color: #F6A21D;
}

.emotion-5 a:hover,
.emotion-5 a:focus {
  border-bottom: solid 0.125rem transparent;
}

@media (max-width: 24.9375rem) {
  .emotion-5 {
    padding: 2.75rem 1rem 0.5rem 1rem;
  }
}

@media (min-width: 25rem) and (max-width: 37.4375rem) {
  .emotion-5 {
    padding: 2.75rem 1rem 0.5rem 1rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-5 {
    padding: calc(2rem - 0.0625rem) 1rem 2rem 1rem;
  }
}

.emotion-7 {
  font-size: 1.125rem;
  line-height: 1.375rem;
  color: #FFFFFF;
  font-weight: 700;
  padding-top: 1rem;
  margin: 0;
}

@media (min-width: 20rem) and (max-width: 37.4375rem) {
  .emotion-7 {
    font-size: 1.25rem;
    line-height: 1.5rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-7 {
    font-size: 1.5rem;
    line-height: 1.75rem;
  }
}

.emotion-7:focus {
  outline: none;
}

.emotion-9 {
  font-size: 0.9375rem;
  line-height: 1.25rem;
  margin-top: 1rem;
  margin-bottom: 1.5rem;
  color: #BEBEBE;
}

@media (min-width: 20rem) and (max-width: 37.4375rem) {
  .emotion-9 {
    font-size: 1rem;
    line-height: 1.375rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-9 {
    font-size: 1rem;
    line-height: 1.375rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-9 {
    margin-top: 1.5rem;
  }
}

.emotion-11 {
  font-size: 0.9375rem;
  line-height: 1.125rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #F6A21D;
  font-weight: 600;
  padding: 0;
  margin: 0;
  list-style-type: none;
}

@media (min-width: 20rem) and (max-width: 37.4375rem) {
  .emotion-11 {
    font-size: 0.9375rem;
    line-height: 1.125rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-11 {
    font-size: 0.875rem;
    line-height: 1.125rem;
  }
}

.emotion-11 li+li {
  margin-top: 0.5rem;
  padding-top: 1rem;
  padding-bottom: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
}

@media (min-width: 37.5rem) {
  .emotion-11 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
    -webkit-box-pack: justify;
    -webkit-justify-content: space-between;
    justify-content: space-between;
  }

  .emotion-11 li+li {
    margin-top: 0;
  }
}

.emotion-13 {
  text-align: center;
  width: 100%;
  word-break: break-word;
}

.emotion-13 button {
  font-size: 0.9375rem;
  line-height: 1.125rem;
  width: 100%;
  min-height: 2.75rem;
  color: #222222;
  font-weight: bold;
  background-color: #FDFDFD;
  border: solid 0.0625rem transparent;
  margin: 0;
  cursor: pointer;
}

@media (min-width: 20rem) and (max-width: 37.4375rem) {
  .emotion-13 button {
    font-size: 0.9375rem;
    line-height: 1.125rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-13 button {
    font-size: 0.875rem;
    line-height: 1.125rem;
  }
}

.emotion-13 button:focus,
.emotion-13 button:hover {
  color: #222222;
  background-color: #F6A21D;
}

.emotion-13 button:hover,
.emotion-13 button:focus {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-13 button:focus-visible {
  outline: 0.1875rem solid #000000;
}

@media (min-width: 25rem) {
  .emotion-13 {
    width: 17.3125rem;
  }
}

.emotion-13.hide {
  width: 2.75rem;
  height: 2.75rem;
  position: absolute;
  top: 0;
  right: 0;
  margin: 0;
  padding: 0;
}

.emotion-13.hide button {
  width: 2.75rem;
  height: 2.75rem;
  cursor: pointer;
  background: none;
  border: none;
}

.emotion-13.hide button:focus::after,
.emotion-13.hide button:hover::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  top: 0;
  bottom: 0;
  border: 0.1875rem solid #FFFFFF;
}

.emotion-13.hide button:focus-visible::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  top: 0;
  bottom: 0;
  border: 0.1875rem solid #000000;
  box-shadow: 0 0 0 0.1875rem #FFFFFF inset;
}

.emotion-13.hide svg {
  color: white;
  fill: currentColor;
  position: absolute;
  top: 0;
  right: 0;
}

.emotion-18 {
  background-color: #B80000;
  height: 2.75rem;
  width: 100%;
  padding: 0 0.5rem;
  position: relative;
  z-index: 1;
}

@media (min-width: 25rem) {
  .emotion-18 {
    height: 3.5rem;
    padding: 0 1rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-18 {
    height: 4rem;
  }
}

.emotion-18 svg {
  fill: currentColor;
}

@media screen and (forced-colors: active) {
  .emotion-18 svg {
    fill: linkText;
  }
}

.emotion-20 {
  height: 100%;
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  max-width: 63rem;
  margin: 0 auto;
}

@media (max-width: 14.9375rem) {
  .emotion-20 {
    display: block;
  }
}

.emotion-22 {
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: relative;
  bottom: 0.125rem;
  padding-top: 0.125rem;
}

.emotion-22:hover,
.emotion-22:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
  border-bottom: 0.25rem solid #FFFFFF;
  margin-bottom: -0.25rem;
}

.emotion-22:focus-visible::after {
  content: '';
  position: absolute;
  top: 0;
  left: -0.1875rem;
  bottom: 0;
  right: -0.1875rem;
  border-top: 0.25rem solid #FFFFFF;
  outline: 0.25rem solid #FFFFFF;
}

.emotion-24 {
  box-sizing: content-box;
  color: #FFFFFF;
  fill: currentColor;
  height: 1.25rem;
}

@media (min-width: 25rem) {
  .emotion-24 {
    height: 1.5rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-24 {
    height: 1.875rem;
  }
}

@media screen and (-ms-high-contrast: active),print {
  .emotion-24 {
    fill: windowText;
  }
}

.emotion-26 {
  -webkit-clip-path: inset(100%);
  clip-path: inset(100%);
  clip: rect(1px, 1px, 1px, 1px);
  height: 1px;
  overflow: hidden;
  position: absolute;
  width: 1px;
  margin: 0;
}

.emotion-27 {
  position: absolute;
  -webkit-clip-path: inset(100%);
  clip-path: inset(100%);
  clip: rect(1px, 1px, 1px, 1px);
  height: 1px;
  width: 1px;
  overflow: hidden;
  padding: 0.75rem 0.5rem;
  background-color: #FFFFFF;
  border: 0.1875rem solid #000;
  color: #333;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 0.9375rem;
  line-height: 1.25rem;
  font-family: ReithSans,Helvetica,Arial,sans-serif;
  font-weight: 400;
  font-style: normal;
}

@media (min-width: 20rem) and (max-width: 37.4375rem) {
  .emotion-27 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-27 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

.emotion-27:focus {
  -webkit-clip-path: none;
  clip-path: none;
  clip: auto;
  height: auto;
  width: auto;
  top: 0;
  left: 0;
}

@media (min-width: 25rem) {
  .emotion-27:focus {
    top: 0.5rem;
  }
}

@media (max-width: 37.4375rem) {
  .emotion-27 {
    padding: 0.5rem;
  }
}

.emotion-29 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  position: relative;
}

.emotion-30 {
  content-visibility: auto;
  contain-intrinsic-size: 33.125rem;
}

@media (min-width: 15rem) {
  .emotion-30 {
    contain-intrinsic-size: 26.563rem;
  }
}

@media (min-width: 20) {
  .emotion-30 {
    contain-intrinsic-size: 23.438rem;
  }
}

@media (min-width: 25rem) {
  .emotion-30 {
    contain-intrinsic-size: 21.875rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-30 {
    contain-intrinsic-size: 17.188rem;
  }
}

.emotion-33 {
  background-color: #B80000;
  height: 2.75rem;
  width: 100%;
  padding: 0 0.5rem;
  border-top: 0.0625rem solid transparent;
  position: relative;
  z-index: 1;
}

@media (min-width: 25rem) {
  .emotion-33 {
    height: 3.5rem;
    padding: 0 1rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-33 {
    height: 4rem;
  }
}

.emotion-33 svg {
  fill: currentColor;
}

@media screen and (forced-colors: active) {
  .emotion-33 svg {
    fill: linkText;
  }
}

.emotion-42 {
  font-size: 0.875rem;
  line-height: 1.125rem;
  font-family: ReithSans,Helvetica,Arial,sans-serif;
  font-style: normal;
  font-weight: 400;
  background-color: #222222;
}

@media (min-width: 20rem) and (max-width: 37.4375rem) {
  .emotion-42 {
    font-size: 0.875rem;
    line-height: 1.125rem;
  }
}

@media (min-width: 37.5rem) {
  .emotion-42 {
    font-size: 0.8125rem;
    line-height: 1rem;
  }
}

@media (max-width: 37.4375rem) {
  .emotion-42 {
    padding: 0 0.5rem;
  }
}

@media (min-width: 25rem) {
  .emotion-42 {
    padding: 0 1rem;
  }
}

.emotion-43 {
  max-width: 63rem;
  margin: 0 auto;
  padding-top: 0.5rem;
}

.emotion-44 {
  border-bottom: 0.0625rem solid #3F3F42;
  -webkit-column-count: 4;
  column-count: 4;
  margin: 0;
  list-style-type: none;
  padding: 0 0 0.5rem;
}

@supports (grid-template-columns: fit-content(200px)) {
  .emotion-44 {
    display: grid;
    grid-auto-flow: column;
  }
}

@media (max-width: 14.9375rem) {
  .emotion-44 {
    grid-auto-flow: row;
    -webkit-column-count: 1;
    column-count: 1;
  }
}

@media (min-width: 15rem) and (max-width: 37.4375rem) {
  .emotion-44 {
    grid-column-gap: 0.5rem;
    grid-template-columns: repeat(2, 1fr);
    -webkit-column-count: 2;
    column-count: 2;
  }
}

@media (min-width: 37.5rem) and (max-width: 62.9375rem) {
  .emotion-44 {
    grid-column-gap: 1rem;
    grid-template-columns: repeat(3, 1fr);
    -webkit-column-count: 3;
    column-count: 3;
  }
}

@media (min-width: 63rem) and (max-width: 79.9375rem) {
  .emotion-44 {
    grid-column-gap: 1rem;
    grid-template-columns: repeat(4, 1fr);
    -webkit-column-count: 4;
    column-count: 4;
  }
}

@media (min-width: 80rem) {
  .emotion-44 {
    grid-column-gap: 1rem;
    grid-template-columns: repeat(5, 1fr);
    -webkit-column-count: 5;
    column-count: 5;
  }
}

.emotion-44>li:first-of-type {
  border-bottom: 0.0625rem solid #3F3F42;
  padding: 0.5rem 0;
  margin-bottom: 0.5rem;
  grid-column: 1/-1;
  width: 100%;
  -webkit-column-span: all;
  column-span: all;
}

@media (min-width: 15rem) and (max-width: 37.4375rem) {
  .emotion-44 {
    grid-template-rows: repeat(5, auto);
  }
}

@media (min-width: 37.5rem) and (max-width: 62.9375rem) {
  .emotion-44 {
    grid-template-rows: repeat(4, auto);
  }
}

@media (min-width: 63rem) and (max-width: 79.9375rem) {
  .emotion-44 {
    grid-template-rows: repeat(3, auto);
  }
}

@media (min-width: 80rem) {
  .emotion-44 {
    grid-template-rows: repeat(3, auto);
  }
}

.emotion-45 {
  min-width: 50%;
  -webkit-column-gap: 1rem;
  column-gap: 1rem;
  break-inside: avoid-column;
}

.emotion-46 {
  font-family: ReithSans,Helvetica,Arial,sans-serif;
  font-style: normal;
  font-weight: 700;
  color: #FFFFFF;
  padding: 0.75rem 0 0.75rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: block;
}

.emotion-46:hover,
.emotion-46:focus {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-53 {
  color: #FFFFFF;
  margin: 0;
  padding: 1rem 0;
}

.emotion-53 a {
  padding: 0;
}

.emotion-54 {
  font-family: ReithSans,Helvetica,Arial,sans-serif;
  font-style: normal;
  font-weight: 700;
  color: #FFFFFF;
  padding: 0.75rem 0 0.75rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: inline;
}

.emotion-54:hover,
.emotion-54:focus {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<div>
  <div
    class="emotion-0"
    id="main-wrapper"
  >
    <header
      role="banner"
    >
      <div>
        <div
          class="emotion-1 emotion-2"
        >
          <div
            class="emotion-3 emotion-4"
            dir="ltr"
          >
            <div
              class="emotion-5 emotion-6"
              dir="ltr"
            >
              <h2
                class="emotion-7 emotion-8"
                dir="ltr"
                tabindex="-1"
              >
                We've updated our Privacy and Cookies Policy
              </h2>
              <p
                class="emotion-9 emotion-10"
                dir="ltr"
              >
                We've made some important changes to our Privacy and Cookies Policy and we want you to know what this means for you and your data.
              </p>
              <ul
                class="emotion-11 emotion-12"
                dir="ltr"
                role="list"
              >
                <li
                  class="emotion-13 emotion-14"
                  dir="ltr"
                >
                  <button
                    data-terms-banner="accept"
                    type="button"
                  >
                    OK
                  </button>
                </li>
                <li
                  class="emotion-13 emotion-14"
                  dir="ltr"
                >
                  <span>
                    <a
                      data-terms-banner="reject"
                      href="https://www.bbc.co.uk/usingthebbc/your-data-matters"
                    >
                      Find out what's changed
                    </a>
                  </span>
                </li>
              </ul>
            </div>
          </div>
        </div>
        <div
          class="emotion-17 emotion-18 emotion-19"
        >
          <div
            class="emotion-20 emotion-21"
          >
            <a
              aria-labelledby="BrandLink-topPage"
              class="focusIndicatorRemove emotion-22 emotion-23"
              href="/news"
              id="topPage"
            >
              <svg
                aria-hidden="true"
                class="emotion-24 emotion-25"
                focusable="false"
                height="32"
                id="brandSvgHeader"
                viewBox="0 0 980 142"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill-rule="evenodd"
                  stroke="#000"
                  stroke-width=".335"
                  style="stroke: #fff;"
                >
                  <path
                    d="M0 0v141.732h141.732V0Zm98.839 100.024a23.16 23.16 0 0 1-10.352 7.917 42.088 42.088 0 0 1-16.153 2.787H40.571V31.004h29.92q13.224 0 20.489 5.296 7.261 5.296 7.263 15.22a18.29 18.29 0 0 1-2.602 9.868 17.917 17.917 0 0 1-7.697 6.634 21.248 21.248 0 0 1 10.786 7.415 20.186 20.186 0 0 1 3.74 12.32 20.663 20.663 0 0 1-3.631 12.267Zm-20.272-38.97a9.358 9.358 0 0 0 3.307-7.638q0-9.476-12.901-9.477H56.724v19.847h12.249q6.287 0 9.594-2.732Zm-7.534 15.22H56.724v21.52h14.093q7.261 0 11.166-2.732a9.01 9.01 0 0 0 3.903-7.861q-.001-10.926-14.853-10.927ZM177.166 0v141.732h141.732V0Zm98.838 100.024a23.16 23.16 0 0 1-10.352 7.917 42.088 42.088 0 0 1-16.153 2.787h-31.763V31.004h29.92q13.224 0 20.489 5.296 7.261 5.298 7.263 15.22a18.29 18.29 0 0 1-2.602 9.868 17.917 17.917 0 0 1-7.697 6.634 21.248 21.248 0 0 1 10.786 7.415 20.187 20.187 0 0 1 3.74 12.32 20.663 20.663 0 0 1-3.631 12.267Zm-20.271-38.97a9.358 9.358 0 0 0 3.307-7.638q0-9.476-12.901-9.477H233.89v19.847h12.249q6.287 0 9.594-2.732Zm-7.535 15.22h-14.309v21.52h14.093q7.261 0 11.166-2.732a9.01 9.01 0 0 0 3.903-7.861q-.001-10.926-14.853-10.927Zm106.133 65.459h141.732V0H354.331Zm101.793-36.057a46.238 46.238 0 0 1-11.478 4.316 58.045 58.045 0 0 1-13.729 1.585 50.05 50.05 0 0 1-17.217-2.787 35.175 35.175 0 0 1-12.941-8.033 34.802 34.802 0 0 1-8.102-12.842 48.987 48.987 0 0 1-2.814-17.104 45.738 45.738 0 0 1 2.926-16.776 35.83 35.83 0 0 1 8.44-12.842 37.18 37.18 0 0 1 13.222-8.197 49.66 49.66 0 0 1 17.274-2.842 55.607 55.607 0 0 1 12.547 1.366 46.64 46.64 0 0 1 10.972 3.989v15.519a36.756 36.756 0 0 0-10.016-4.645 40.183 40.183 0 0 0-11.365-1.585 29.66 29.66 0 0 0-14.01 3.06 20.959 20.959 0 0 0-8.89 8.853 29.58 29.58 0 0 0-3.094 14.098 30.845 30.845 0 0 0 2.982 14.154 20.367 20.367 0 0 0 8.664 8.907 28.727 28.727 0 0 0 13.785 3.06 42.724 42.724 0 0 0 22.844-6.448Zm162.072 18.34q-11.596-18.582-23.49-37.242-11.897-18.656-23.49-36.2v73.442h-22.003V17.718h23.192q12.042 17.99 23.713 36.052 11.668 18.063 23.266 36.646V17.718h22.003v106.298Zm44.258-106.298h73.145v19.029h-50.25V59.79h44.601V78.82h-44.601v26.166h53.075v19.03h-75.97Zm177.915 106.298q-5.055-17.987-9.886-35.978-4.833-17.987-9.441-35.978-4.46 17.99-9.217 35.978-4.76 17.99-9.812 35.978H777.78Q770.198 98 763.21 71.386q-6.99-26.61-13.231-53.67h24.976q3.715 20.666 7.88 41.33 4.16 20.668 8.771 40.587 9.959-38.802 18.881-77.456h22.3q4.461 19.032 9.143 37.985 4.683 18.955 9.589 37.836 4.46-19.625 8.623-39.843 4.16-20.216 7.73-40.438h24.531q-6.244 27.06-13.231 53.67-6.99 26.613-14.57 52.629Zm95.37 2.23a78.37 78.37 0 0 1-17.32-1.933 66.24 66.24 0 0 1-16.428-6.096V97.7a63.933 63.933 0 0 0 33.748 9.664q10.109 0 15.387-3.42a10.75 10.75 0 0 0 5.278-9.514 9.755 9.755 0 0 0-2.601-7.211 18.56 18.56 0 0 0-6.913-4.163 99.45 99.45 0 0 0-9.663-2.9 110.65 110.65 0 0 1-15.462-4.756 45.068 45.068 0 0 1-11.299-6.32 23.846 23.846 0 0 1-6.987-8.77 28.942 28.942 0 0 1-2.379-12.265 29.446 29.446 0 0 1 5.129-17.395 32.716 32.716 0 0 1 14.496-11.225q9.366-3.938 22.3-3.94a72.223 72.223 0 0 1 16.651 1.934 67.688 67.688 0 0 1 14.718 5.204v20.219a56.134 56.134 0 0 0-14.792-6.244 63.276 63.276 0 0 0-16.577-2.231q-9.07 0-14.049 3.122a9.935 9.935 0 0 0-4.981 8.92 9.093 9.093 0 0 0 2.379 6.616 17.36 17.36 0 0 0 6.765 4.014q4.383 1.561 10.332 3.048 7.583 1.935 14.124 4.237a45.864 45.864 0 0 1 11.521 5.872 24.459 24.459 0 0 1 7.731 9.069 30.061 30.061 0 0 1 2.75 13.529q0 10.558-5.203 18.063a32.29 32.29 0 0 1-15.016 11.447q-9.813 3.939-23.639 3.94Z"
                  />
                </g>
              </svg>
              <span
                class="emotion-26"
                id="BrandLink-topPage"
              >
                BBC News
              </span>
            </a>
            <a
              class="focusIndicatorRemove emotion-27 emotion-28"
              dir="ltr"
              href="#content"
            >
              <div>
                Skip to content
              </div>
            </a>
          </div>
        </div>
      </div>
    </header>
    <div
      class="emotion-29"
    >
      <h2>
        Child element
      </h2>
    </div>
    <footer
      class="emotion-30 emotion-31"
      role="contentinfo"
    >
      <div
        class="emotion-17 emotion-33 emotion-19"
      >
        <div
          class="emotion-20 emotion-21"
        >
          <a
            aria-labelledby="BrandLink-footer"
            class="focusIndicatorRemove emotion-22 emotion-23"
            href="/news"
            id="footer"
          >
            <svg
              aria-hidden="true"
              class="emotion-24 emotion-25"
              focusable="false"
              height="32"
              id="brandSvgFooter"
              viewBox="0 0 980 142"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill-rule="evenodd"
                stroke="#000"
                stroke-width=".335"
                style="stroke: #fff;"
              >
                <path
                  d="M0 0v141.732h141.732V0Zm98.839 100.024a23.16 23.16 0 0 1-10.352 7.917 42.088 42.088 0 0 1-16.153 2.787H40.571V31.004h29.92q13.224 0 20.489 5.296 7.261 5.296 7.263 15.22a18.29 18.29 0 0 1-2.602 9.868 17.917 17.917 0 0 1-7.697 6.634 21.248 21.248 0 0 1 10.786 7.415 20.186 20.186 0 0 1 3.74 12.32 20.663 20.663 0 0 1-3.631 12.267Zm-20.272-38.97a9.358 9.358 0 0 0 3.307-7.638q0-9.476-12.901-9.477H56.724v19.847h12.249q6.287 0 9.594-2.732Zm-7.534 15.22H56.724v21.52h14.093q7.261 0 11.166-2.732a9.01 9.01 0 0 0 3.903-7.861q-.001-10.926-14.853-10.927ZM177.166 0v141.732h141.732V0Zm98.838 100.024a23.16 23.16 0 0 1-10.352 7.917 42.088 42.088 0 0 1-16.153 2.787h-31.763V31.004h29.92q13.224 0 20.489 5.296 7.261 5.298 7.263 15.22a18.29 18.29 0 0 1-2.602 9.868 17.917 17.917 0 0 1-7.697 6.634 21.248 21.248 0 0 1 10.786 7.415 20.187 20.187 0 0 1 3.74 12.32 20.663 20.663 0 0 1-3.631 12.267Zm-20.271-38.97a9.358 9.358 0 0 0 3.307-7.638q0-9.476-12.901-9.477H233.89v19.847h12.249q6.287 0 9.594-2.732Zm-7.535 15.22h-14.309v21.52h14.093q7.261 0 11.166-2.732a9.01 9.01 0 0 0 3.903-7.861q-.001-10.926-14.853-10.927Zm106.133 65.459h141.732V0H354.331Zm101.793-36.057a46.238 46.238 0 0 1-11.478 4.316 58.045 58.045 0 0 1-13.729 1.585 50.05 50.05 0 0 1-17.217-2.787 35.175 35.175 0 0 1-12.941-8.033 34.802 34.802 0 0 1-8.102-12.842 48.987 48.987 0 0 1-2.814-17.104 45.738 45.738 0 0 1 2.926-16.776 35.83 35.83 0 0 1 8.44-12.842 37.18 37.18 0 0 1 13.222-8.197 49.66 49.66 0 0 1 17.274-2.842 55.607 55.607 0 0 1 12.547 1.366 46.64 46.64 0 0 1 10.972 3.989v15.519a36.756 36.756 0 0 0-10.016-4.645 40.183 40.183 0 0 0-11.365-1.585 29.66 29.66 0 0 0-14.01 3.06 20.959 20.959 0 0 0-8.89 8.853 29.58 29.58 0 0 0-3.094 14.098 30.845 30.845 0 0 0 2.982 14.154 20.367 20.367 0 0 0 8.664 8.907 28.727 28.727 0 0 0 13.785 3.06 42.724 42.724 0 0 0 22.844-6.448Zm162.072 18.34q-11.596-18.582-23.49-37.242-11.897-18.656-23.49-36.2v73.442h-22.003V17.718h23.192q12.042 17.99 23.713 36.052 11.668 18.063 23.266 36.646V17.718h22.003v106.298Zm44.258-106.298h73.145v19.029h-50.25V59.79h44.601V78.82h-44.601v26.166h53.075v19.03h-75.97Zm177.915 106.298q-5.055-17.987-9.886-35.978-4.833-17.987-9.441-35.978-4.46 17.99-9.217 35.978-4.76 17.99-9.812 35.978H777.78Q770.198 98 763.21 71.386q-6.99-26.61-13.231-53.67h24.976q3.715 20.666 7.88 41.33 4.16 20.668 8.771 40.587 9.959-38.802 18.881-77.456h22.3q4.461 19.032 9.143 37.985 4.683 18.955 9.589 37.836 4.46-19.625 8.623-39.843 4.16-20.216 7.73-40.438h24.531q-6.244 27.06-13.231 53.67-6.99 26.613-14.57 52.629Zm95.37 2.23a78.37 78.37 0 0 1-17.32-1.933 66.24 66.24 0 0 1-16.428-6.096V97.7a63.933 63.933 0 0 0 33.748 9.664q10.109 0 15.387-3.42a10.75 10.75 0 0 0 5.278-9.514 9.755 9.755 0 0 0-2.601-7.211 18.56 18.56 0 0 0-6.913-4.163 99.45 99.45 0 0 0-9.663-2.9 110.65 110.65 0 0 1-15.462-4.756 45.068 45.068 0 0 1-11.299-6.32 23.846 23.846 0 0 1-6.987-8.77 28.942 28.942 0 0 1-2.379-12.265 29.446 29.446 0 0 1 5.129-17.395 32.716 32.716 0 0 1 14.496-11.225q9.366-3.938 22.3-3.94a72.223 72.223 0 0 1 16.651 1.934 67.688 67.688 0 0 1 14.718 5.204v20.219a56.134 56.134 0 0 0-14.792-6.244 63.276 63.276 0 0 0-16.577-2.231q-9.07 0-14.049 3.122a9.935 9.935 0 0 0-4.981 8.92 9.093 9.093 0 0 0 2.379 6.616 17.36 17.36 0 0 0 6.765 4.014q4.383 1.561 10.332 3.048 7.583 1.935 14.124 4.237a45.864 45.864 0 0 1 11.521 5.872 24.459 24.459 0 0 1 7.731 9.069 30.061 30.061 0 0 1 2.75 13.529q0 10.558-5.203 18.063a32.29 32.29 0 0 1-15.016 11.447q-9.813 3.939-23.639 3.94Z"
                />
              </g>
            </svg>
            <span
              class="emotion-26"
              id="BrandLink-footer"
            >
              BBC News
            </span>
          </a>
        </div>
      </div>
      <div
        class="emotion-42"
      >
        <div
          class="emotion-43"
        >
          <ul
            class="emotion-44"
            role="list"
          >
            <li
              class="emotion-45"
            >
              <a
                class="focusIndicatorInvert emotion-46"
                href="https://www.bbc.com/news/help-41670342"
              >
                Why you can trust the BBC
              </a>
            </li>
            <li
              role="listitem"
            >
              <a
                class="focusIndicatorInvert emotion-46"
                href="https://www.bbc.com/terms"
              >
                Terms of Use
              </a>
            </li>
            <li
              role="listitem"
            >
              <a
                class="focusIndicatorInvert emotion-46"
                href="https://www.bbc.co.uk/aboutthebbc/"
              >
                About the BBC
              </a>
            </li>
            <li
              role="listitem"
            >
              <a
                class="focusIndicatorInvert emotion-46"
                href="https://www.bbc.com/privacy/"
              >
                Privacy Policy
              </a>
            </li>
            <li
              role="listitem"
            >
              <a
                class="focusIndicatorInvert emotion-46"
                href="https://www.bbc.com/usingthebbc/cookies/"
              >
                Cookies
              </a>
            </li>
            <li
              role="listitem"
            >
              <a
                class="focusIndicatorInvert emotion-46"
                href="https://www.bbc.com/accessibility/"
              >
                Accessibility Help
              </a>
            </li>
            <li
              role="listitem"
            >
              <a
                class="focusIndicatorInvert emotion-46"
                href="https://www.bbc.com/contact/"
              >
                Contact the BBC
              </a>
            </li>
            <li
              role="listitem"
            />
          </ul>
          <p
            class="emotion-53"
          >
            <span
              lang="en-GB"
            >
              ©
               
            </span>
            2024 BBC. The BBC is not responsible for the content of external sites.
             
            <a
              class="focusIndicatorInvert emotion-54"
              href="https://www.bbc.co.uk/editorialguidelines/guidance/feeds-and-links"
            >
              Read about our approach to external linking.
            </a>
          </p>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
